src/Library/demos/Box/main.blp

```diff
@@ -0,0 +1,109 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.Clamp {
+  Box {
+    orientation: vertical;
+
+    Label {
+      label: "Box";
+      margin-top: 12;
+      margin-bottom: 12;
+
+      styles [
+        "title-1",
+      ]
+    }
+
+    Label {
+      label: "A widget that arranges its child widgets into a single row or column";
+      margin-bottom: 18;
+    }
+
+    Box {
+      orientation: vertical;
+
+      Box {
+        hexpand: true;
+        orientation: vertical;
+        spacing: 12;
+
+        Box {
+          halign: center;
+          spacing: 18;
+          Button button_add{
+            label: "Add Item";
+            styles [
+              "suggested-action",
+              "pill",
+            ]
+          }
+          Button button_remove{
+            label: "Remove Item";
+            styles [
+              "destructive-action",
+              "pill",
+            ]
+          }
+        }
+
+        Box {
+          halign: center;
+          spacing: 18;
+
+          CheckButton highlight {
+            label: "Highlight Box";
+          }
+
+          CheckButton toggle_orient {
+            label: "Toggle Orientation";
+          }
+
+          CheckButton homogeneous {
+            label: "Homogeneous";
+          }
```

Comment:
This won't do much in this demo since all items already have the same size anyway.

I'd suggest removing it

Reply:
When there's space available, it will force the children to take all the available space and share it equally right?

![image](https://user-images.githubusercontent.com/58233418/224538170-b387e8cd-1714-4be8-bea4-c8385e9838ca.png)

![image](https://user-images.githubusercontent.com/58233418/224538218-452ce74e-dea5-4071-8a0b-937abb386c7a.png)

 If you still think that it's not very useful to have I can remove it 👍🏾  

I understand that homogeneous is generally intended to be used with children that are taking uneven amounts of space and force them to take space equally.

Reply:
`homogeneous` only guarantees that children will have the same dimensions. Perhaps we could keep it if `Append Item` / `Prepend Item` would create elements of slightly different dimensions 

but it might get confusing

let's leave it out for now and in a second iteration – we can make the children editable – then it will make sense to have a "homogeneous" control